packages/components/skeleton-loader/src/skeleton-loader.ts

```diff
@@ -0,0 +1,50 @@
+import { buildProps } from '@puik/utils'
+import type { ExtractPropTypes } from 'vue'
+import type SkeletonLoader from './skeleton-loader.vue'
+
+export const skeletonLoaderVariants = [
+  'h1',
+  'h2',
+  'h3',
+  'h4',
+  'h5',
+  'h6',
+  'jumbotron',
+  'mega-jumbotron',
+  'text-small',
+  'text-medium',
+  'text-large',
+  'badge',
+  'tag',
+  'image',
+  'graph',
+  'video',
+]
+
+export const skeletonLoaderProps = buildProps({
+  variant: {
+    type: String,
```

Comment:
Using `PropType` here can be a good thing to provide strong typing when using the component